### PR TITLE
pythonPackages.pytest-randomly: 3.6.0 -> 3.10.1

### PR DESCRIPTION
**Pull request by sternenseemann**
https://github.com/pytest-dev/pytest-randomly/blob/main/HISTORY.rst#380-2021-05-10

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sternenseemann** [comment]
@ofborg build python310Packages.pytest-randomly python3Packages.httplib2 python3Packages.fonttools

**Comment by sternenseemann** [comment]
@ofborg build python3Packages.httplib2 python3Packages.fonttools

**Comment by sternenseemann** [comment]
Seems like python310Packages.pytest is broken.

**Comment by sternenseemann** [comment]
@ofborg eval

**Comment by sternenseemann** [comment]
@ofborg build python3Packages.httplib2 python3Packages.fonttools python3Packages.mlrose

**Comment by sternenseemann** [comment]
@jonringer I think this is good to go. I tested all users of pytest-randomly on my branch (which is based against the merge-base of staging and master) and everything works there.

Unfortunately testing on staging isn't possible as the build fails for unrelated reasons (the reason being gfortran).

**Comment by jonringer** [comment]
> Maybe we should merge this into python-unstable as well?

sounds good

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -965,8 +965,6 @@ in {
 
   backports-datetime-fromisoformat = callPackage ../development/python-modules/backports-datetime-fromisoformat { };
 
-  backports-entry-points-selectable = callPackage ../development/python-modules/backports-entry-points-selectable { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-packages.nix:968]: This should get an entry in python-aliases.

Comment (jonringer) [reply pkgs/top-level/python-packages.nix:968]: forgot that now exists

Comment (sternenseemann) [reply pkgs/top-level/python-packages.nix:968]: Done.

**Comment by sternenseemann** [comment]
Retargeted now.

@ofborg build python3Packages.mlrose python3Packages.httplib2 python3Packages.fonttools

**Comment by sternenseemann** [comment]
Can we restore the `python-unstable` branch? I can neither reopen nor retarget this PR without it unfortunately.

**Comment by vcunat** [comment]
Oh, I'm sorry.  I assumed it was a one-merge branch and clicked the button suggested by GitHub.  It's quite easy to leave lots of garbage branches behind.

**Comment by sternenseemann** [comment]
We can probably delete it after retargeting the two PRs that were still open targeted at it to staging.

**Comment by vcunat** [comment]
Hydra jobsets can be easily retargeted to different branches/repos/... **if** you'll want that.

**Comment by sternenseemann** [comment]
@ofborg build python3Packages.mlrose python3Packages.httplib2 python3Packages.fonttools

**Comment by sternenseemann** [comment]
@ofborg python3Packages.mlrose python3Packages.httplib2 python3Packages.fonttools

**Pull request merged**